opal/corelib/range.rb

```diff
@@ -85,16 +93,26 @@ def exclude_end?
     @exclude
   end
 
-  alias :first :begin
+  def first(n=(n_specified=true; nil))
```

Comment:
No that `n` can't be `null` ever when passed a legit ruby value (unless explicitly called with backticks like ```first(`null`)```)?